scripts/buildTs.js

```diff
@@ -23,6 +23,24 @@ const packagesWithTs = packages.filter(p =>
   fs.existsSync(path.resolve(p, 'tsconfig.json'))
 );
 
+packagesWithTs.forEach(pkgDir => {
+  const pkg = require(pkgDir + '/package.json');
+
+  if (!pkg.types) {
+    throw new Error(`Package ${pkg.name} is missing \`types\` field`);
+  }
+
+  if (!pkg.typesVersions) {
```

Comment:
Should this also check the content? Just to make sure they are all the same since they do all use the same downleveling build process.

Reply:
meh 😀 This is more "did you _really_ mess up" than any thorough check